diff --git a/AppBundle/Controller/MightyMouseController.php b/AppBundle/Controller/MightyMouseController.php
index db66838..7401bae 100644
--- a/AppBundle/Controller/MightyMouseController.php
+++ b/AppBundle/Controller/MightyMouseController.php
@@ -18,6 +18,8 @@ class MightyMouseController  extends ContainerAware
             array('quote' => 'Here I come to save the day!')
         );
 
+        dump($this->container->get('twig'));
+
         return new Response($html);
     }
 }
diff --git a/AppKernel.php b/AppKernel.php
index babd47c..25af726 100644
--- a/AppKernel.php
+++ b/AppKernel.php
@@ -18,6 +18,7 @@ class AppKernel extends Kernel
 
         if ($this->getEnvironment() == 'dev') {
             $bundles[] = new \Symfony\Bundle\WebProfilerBundle\WebProfilerBundle();
+            $bundles[] = new \Symfony\Bundle\DebugBundle\DebugBundle();
         }
 
         return $bundles;
