diff --git a/AppBundle/Controller/DefaultController.php b/AppBundle/Controller/DefaultController.php
index a69979a..63d31d2 100644
--- a/AppBundle/Controller/DefaultController.php
+++ b/AppBundle/Controller/DefaultController.php
@@ -18,6 +18,8 @@ class DefaultController extends ContainerAware
             array('random' => rand(1, 100))
         );
 
+        dump($this->container->get('twig'));
+
         return new Response($html);
     }
 }
diff --git a/AppKernel.php b/AppKernel.php
index b031b37..52126fb 100644
--- a/AppKernel.php
+++ b/AppKernel.php
@@ -18,6 +18,7 @@ class AppKernel extends Kernel
 
         if ($this->getEnvironment() == 'dev') {
             $bundles[] = new \Symfony\Bundle\WebProfilerBundle\WebProfilerBundle();
+            $bundles[] = new \Symfony\Bundle\DebugBundle\DebugBundle();
         }
 
         return $bundles;
