diff --git a/.env b/.env
index a218ae2..7f787ac 100644
--- a/.env
+++ b/.env
@@ -1,3 +1,6 @@
 # basic setup
-SYMFONY_ENV=prod
-SYMFONY_DEBUG=0
+SYMFONY_ENV=dev
+SYMFONY_DEBUG=1
+
+# parameters (SYMFONY__ is mapped to parameters)
+SYMFONY__SECRET=CHANGEME
diff --git a/.env.example b/.env.example
index b51575d..7f787ac 100644
--- a/.env.example
+++ b/.env.example
@@ -1,3 +1,6 @@
 # basic setup
 SYMFONY_ENV=dev
 SYMFONY_DEBUG=1
+
+# parameters (SYMFONY__ is mapped to parameters)
+SYMFONY__SECRET=CHANGEME
diff --git a/config/config.yml b/config/config.yml
index cb330be..bd9e728 100644
--- a/config/config.yml
+++ b/config/config.yml
@@ -1,5 +1,5 @@
 framework:
-    secret: ABC123
+    secret: %secret%
     templating:
         engines: [twig]
     profiler:
