diff --git a/.env b/.env
index 7f787ac..bae39dd 100644
--- a/.env
+++ b/.env
@@ -4,3 +4,4 @@ SYMFONY_DEBUG=1
 
 # parameters (SYMFONY__ is mapped to parameters)
 SYMFONY__SECRET=CHANGEME
+SYMFONY__LOG_ACTION_LEVEL=debug
diff --git a/.env.example b/.env.example
index 7f787ac..bae39dd 100644
--- a/.env.example
+++ b/.env.example
@@ -4,3 +4,4 @@ SYMFONY_DEBUG=1
 
 # parameters (SYMFONY__ is mapped to parameters)
 SYMFONY__SECRET=CHANGEME
+SYMFONY__LOG_ACTION_LEVEL=debug
diff --git a/config/config.yml b/config/config.yml
index 40eee61..2f3e565 100644
--- a/config/config.yml
+++ b/config/config.yml
@@ -11,7 +11,7 @@ monolog:
     handlers:
         main:
             type:         fingers_crossed
-            action_level: debug
+            action_level: %log_action_level%
             handler:      nested
         nested:
             type:  stream
