diff --git a/AppBundle/Controller/MightyMouseController.php b/AppBundle/Controller/MightyMouseController.php
index 2a5b756..db66838 100644
--- a/AppBundle/Controller/MightyMouseController.php
+++ b/AppBundle/Controller/MightyMouseController.php
@@ -2,16 +2,22 @@
 
 namespace AppBundle\Controller;
 
+use Symfony\Component\DependencyInjection\ContainerAware;
 use Symfony\Component\HttpFoundation\Response;
 use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
 
-class MightyMouseController
+class MightyMouseController  extends ContainerAware
 {
     /**
      * @Route("/")
      */
     public function rescueAction()
     {
-        return new Response('Here I come to save the day!');
+        $html = $this->container->get('twig')->render(
+            'mighty_mouse/rescue.html.twig',
+            array('quote' => 'Here I come to save the day!')
+        );
+
+        return new Response($html);
     }
 }
diff --git a/Resources/views/base.html.twig b/Resources/views/base.html.twig
new file mode 100644
index 0000000..9089212
--- /dev/null
+++ b/Resources/views/base.html.twig
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8" />
+        <title>{% block title %}Welcome!{% endblock %}</title>
+        {% block stylesheets %}{% endblock %}
+        <link rel="icon" type="image/x-icon" href="{{ asset('favicon.ico') }}" />
+    </head>
+    <body>
+        {% block body %}{% endblock %}
+        {% block javascripts %}{% endblock %}
+    </body>
+</html>
\ No newline at end of file
diff --git a/Resources/views/mighty_mouse/rescue.html.twig b/Resources/views/mighty_mouse/rescue.html.twig
new file mode 100644
index 0000000..13ab389
--- /dev/null
+++ b/Resources/views/mighty_mouse/rescue.html.twig
@@ -0,0 +1,5 @@
+{% extends 'base.html.twig' %}
+
+{% block body %}
+    <h1>{{ quote }}</h1>
+{% endblock %}
